array_parser.ts

```diff
@@ -20,19 +20,23 @@
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
-export function parseArray(source: string, transform: Function | undefined) {
+
+// deno-lint-ignore no-explicit-any
+type Transformer = (value: string) => any;
```

Comment:
Hmm, I just ignored these in #175., but this looks like a better way to handle. :) 

Could you rebase over the latest? (and remove the .github/workflows/ci.yml change).

Sorry for slowness in review.

Reply:
Done!

Reply:
Great! Thanks as always @uki00a !